src/overlays/actors/ovl_Bg_Hidan_Firewall/z_bg_hidan_firewall.c

```diff
@@ -21,23 +22,239 @@ const ActorInit Bg_Hidan_Firewall_InitVars =
     (ActorFunc)BgHidanFirewall_Update,
     NULL,
 };
-*/
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_Bg_Hidan_Firewall/BgHidanFirewall_Init.s")
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_Bg_Hidan_Firewall/BgHidanFirewall_Destroy.s")
+static ColliderCylinderInit cylinderInitData =
+{
+    0x0A, 0x11, 0x00, 0x09,
+    0x20, 0x01, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00,
+    0x20000000,
+    0x01, 0x04, 0x00, 0x00,
+    0xFFCFFFFF,
+    0x00, 0x00, 0x00, 0x00,
+    0x19, 0x00, 0x01, 0x00,
+    0x001E,
+    0x0053,
+    0x00000000000000000000,
+};
+
+static SubActor98Init actor98InitData =
+{
+    0x01, 0x0050, 0x0064, 0xFF,
+};
+
+u32 D_80886D04[] = {
+    0x06015D20, 
+    0x06016120, 
+    0x06016520, 
+    0x06016920, 
+    0x06016D20, 
+    0x06017120, 
+    0x06017520, 
+    0x06017920, 
+    0x00000000, 
+    0x00000000, 
+    0x00000000,
+};
+
```

Comment:
we should leave the type as UNK_PTR until we know what these segment addresses are. Also I'm fairly sure the two null entries at the end are just padding and dont need to be included